R/stan_betareg.fit.R

```diff
@@ -24,7 +24,8 @@ stan_betareg.fit <- function(x, y, z = NULL,
                              link.phi = NULL, ...,
                              prior = normal(), prior_intercept = normal(),
                              prior_z = normal(), prior_intercept_z = normal(),
-                             prior_ops = prior_options(), prior_PD = FALSE, 
+                             prior_dispersion = cauchy(0, 5),
+                             prior_ops = NULL, prior_PD = FALSE, 
```

Comment:
Got it. d275d45 has `prior_ops` removed and adjusts `summarize_betareg_prior` accordingly. (Also had to update some tests.)